ajax/libs/signalr.js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "signalr.js",
+  "version": "1.0.0",
+  "filename": "jquery.signalR.js",
+  "description": "Compiled and distributed version of SignalR Client JS scripts.",
+  "homepage": "https://github.com/CreativeColon/signalR.js",
+  "license": "Apache-2.0",
+  "keywords": [
+    "signalr",
+    "chat"
+  ],
+  "author": "Rohith Kumbharkar",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CreativeColon/signalR.js.git"
+  },
+  "dependencies": {
+    "jquery": "^1.6.4"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/CreativeColon/signalR.js.git",
```

Comment:
The official source does not contain compiled files. That's the reason I've created this repo